README.md

```diff
@@ -984,6 +984,10 @@ in inheritance.
   is a hybrid of `Array`'s intuitive inter-operation facilities and
   `Hash`'s fast lookup.
 
+*  Prefer `map` over `collect`, `detect` over `find`, and `reduce` over
```

Comment:
@schleyfox seemed to feel that there was a reason why `map` was clearly superior to `collect` in the original PR, but I don't know what the rationale was. 

I think, per @BMorearty's comments, the `inject`/`reduce` distinction is the one that people have the softest preferences around. I like `reduce` a little more because it's a clear parallel to a programming idiom in other languages, whereas I'm not aware of `inject` having this meaning outside of Ruby.